### ISSUE TITLE
[Feature Request] Checkbox/white/blacklist for anonymising filenames for AnonymiseFileNames

### ISSUE THREAD

**Issue by DyonR** [Vendicated/Vencord#740]
### What is it that you'd like to see?

When sending a file with AnonymiseFileNames enabled, I think it would be nice to have a 'checkbox' like "Spoiler Attachment" available for when do _not_ want to send a file with an anonymised name.  
If I want to share a mp3 file to a friend, it's kinda annoying if the filename changes, because of the anonymised name it's hard to identity who the artist and what the song title is for example. Same with some documents.  
This is how Discord has 'checkboxes' to mark an attachment as Spoiler, if next to that an icon could be added to Enable/Disable filename anonymisation, that would be great.
![image](https://user-images.githubusercontent.com/16341409/228930875-32af9411-cde2-4657-b640-631dacb3d2ee.png)
  
My suggestion for an icon would be something like an 'incognito icon'.  
In the settings of AnonymiseFileNames it would be great to have an option to have it always on by default, so the checkbox disables it for the file, or off by default, so when selecting the checkbox only then it anonymises the filename.  
  
Another suggestion, which maybe should be an different issue would be to have a white/blacklist to have certain extensions always be enabled/disabled by default. So then you could whitelist *.mp3 or *.pdf for example.

### Request Agreement

- [X] This is not a plugin request

**Comment by Vendicated** [comment]
#553

**Comment by DyonR** [comment]
My bad, I used search but somehow didn't realise this was an open PR. Issue can be closed I guess, unless the white/blacklist feature is still desired

**Comment by Vendicated** [comment]
No worries and there's no need to close it since it's not a bad thing to have an issue tracking it. As for white/blacklist, that sounds like a good idea but is blocked by #696 for now
